### ISSUE TITLE
Links to plays on home page

### ISSUE THREAD

**Issue by fdsimms** [lilykriopelle/THMSite#8]
It would be nice if users could click on the photos on the home page and be taken straight to the show pages for the respective piece.

**Comment by lilykriopelle** [comment]
@fdsimms totally agree, you mean like what we have on the retreat/electra page?  we will replace this splash page when we announce our next season, so am hesitant to update it bc it's just filler

**Comment by fdsimms** [comment]
@lilykriopelle yes, exactly. It's probably something to wait on if you're going to update things eventually anyway.

**Comment by lilykriopelle** [comment]
@fdsimms – got it.  closing this now and making a new issue that's just for replacing the splash page w/something better.
